### DISCUSSION TITLE
Eddie Jauode Wikipedia

### DISCUSSION THREAD

**Discussion by ghost** [EddieHubCommunity/support#1521] [Show and tell]
Check it here: https://en.wikipedia.org/wiki/Draft:Eddie_Jaoude

**Comment by ghost** [comment]
https://en.wikipedia.org/wiki/Wikipedia:WikiProject_Articles_for_creation/Reviewing_instructions

Review It With These Guidelines

**Reply by Vyvy-vi** [reply]
I think there's some consideration needed before writing a wiki page. (esp. with respect to Wiki's ethics and values)
There is also a need to make the content in there more encyclopaedic and informative, rather than promotional

(Moreover IMHO, the organisation/individual on which a wiki page is made, should not be involved in any way with the contents of the wiki page)

**Comment by ghost** [comment]
Create your own Wikipedia page https://en.wikipedia.org/wiki/Wikipedia:How_to_create_a_page

**Comment by aminoxix** [comment]
🚀  But, I like to suggest that, if one's really wanna write Wiki pages, then it must be on Community i.e, **EddieHub**; so we got more contents to add-on, well, we can also attach a sub page/topic for person i.e, Eddie  & all other folks..

Hope you'll understand my concern!  :)

**Reply by Vyvy-vi** [reply]
hmm, that's true _however_ I'm not sure if an org should make their own Wikipedia page(considering that the page then potentially turns into an advert... of _sorts_)

However, for now this content isn't wasted. Maybe this can be used for the Eddiehub website, or maybe we can make an eddiehub-wiki, for all things eddiehub :)

**Reply by aminoxix** [reply]
True 💯 @Vyvy-vi 
_We ain't think any content is waste in community_

**Comment by ghost** [comment]
The page has been not approved 😦 😭 Thus marking this as answered

**Reply by eddiejaoude** [reply]
Thank you for trying @krishdevdb 👍 next time we try again 🤓
